browser/src/control/Control.ServerAuditDialog.ts

```diff
@@ -215,6 +218,17 @@ class ServerAuditDialog {
 							entries: entries,
 							enabled: entries.length > 0,
 						},
+						!hasErrors
+							? {
+									id: 'auditsuccess',
+									type: 'fixedtext',
+									text: _('No issues found'),
+								}
+							: {
+									id: 'auditerror',
+									type: 'fixedtext',
+									text: `${countErrors} issue${countErrors > 1 ? 's' : ''} found`,
```

Comment:
We need translatable string, we do that by using special wrapping function: `_('string to translate')`.
It has to be static text, any dynamic content should be provided by replacement of a variable mark like eg. `%COUNT%`.
If you need both single and plural form, please use 2 separate strings, not combine that in JS as it will not work with other languages.

example:
```
if (count === 1)
      return _('1 issue found');
else
      return _('%COUNT issues found').replaceAll('%COUNT%', number);
```

I'm not sure if it is exactly %var% maybe different style, please check within a code for examples

Reply:
Please checkout the new changes